roles/pulp/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+pulp_image: pulp/pulp:stable
+pulp_ports:
+  - "8080:80"
+pulp_volumes:
+  - pulp_settings.volume:/etc/pulp:Z
+  - pulp_storage.volume:/var/lib/pulp:Z
+  - pulp_pgsql.volume:/var/lib/pgsql:Z
+  - pulp_containers.volume:/var/lib/containers:Z
```

Comment:
I learned something today. Adding the `.volume` causes quadlets to treat this as a service providing the volume and not just a simple filesystem volume. This then results in those services being required and failing, here is the output from the generated service file:

```
[Unit]
Wants=network-online.target
After=network-online.target
Description=Pulp
After=network.target
SourcePath=/etc/containers/systemd/pulp.container
RequiresMountsFor=%t/containers
Requires=pulp_settings-volume.service
After=pulp_settings-volume.service
Requires=pulp_storage-volume.service
After=pulp_storage-volume.service
Requires=pulp_pgsql-volume.service
After=pulp_pgsql-volume.service
Requires=pulp_containers-volume.service
After=pulp_containers-volume.service
```

TLDR; Drop the `.volume`